libs/shared/src/consts/providers/channels/email.ts

```diff
@@ -15,6 +16,14 @@ import { ChannelTypeEnum } from '../../../entities/message-template';
 import { EmailProviderIdEnum } from '../provider.enum';
 
 export const emailProviders: IProviderConfig[] = [
+  {
+    id: EmailProviderIdEnum.MSGraphAPI,
+    displayName: 'MS Graph API',
+    channel: ChannelTypeEnum.EMAIL,
+    credentials: msGraphConfigBase,
+    docReference: '',
```

Comment:
This is very important for us or any contributor, to quickly reach the documentation.